src/components/DownloadButtons/DownloadButtons.module.css

```diff
@@ -8,7 +8,7 @@
   gap: 8px;
 }
 
-.zipBtnWapper {
+.zipBtnWrapper {
```

Comment:
How do you feel about this abbreviation of Button? I think we're generally not using such abbreviations in our naming.

Reply:
I was not too happy about it, and since you mention it now, let me rename them!